client/src/components/files/explorer/FileListDisplay.vue

```diff
@@ -30,20 +30,52 @@
               :indeterminate="someSelected && !allSelected"
             />
           </ion-label>
-          <ion-text class="folder-list-header__label cell-title ion-text-nowrap label-name">
+          <ion-text
+            class="folder-list-header__label cell-title ion-text-nowrap label-name"
+            @click="onHeaderSortChange(SortProperty.Name)"
+            :class="{ 'label-name-sorted': currentSortProperty === SortProperty.Name }"
+          >
             {{ $msTranslate('FoldersPage.listDisplayTitles.name') }}
+            <ion-icon
+              :icon="currentSortOrder ? arrowDown : arrowUp"
```

Comment:
This seems reversed: Ascending has an arrow down and descending has an arrow up, the opposite of what's displayed in the sorter.